src/phases/execution/TimeSeriesUpdatesAndDeletes.yml

```diff
@@ -0,0 +1,132 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/server-execution"
+Description: |
+  Set up 1000 independent sensors which will each have 100 buckets, and each bucket has 100
+  measurements.
+
+  The buckets have the meta field, 'sensorId', with values from 0 to 999, each of which has 100
+  measurements with timestamps within an hour with 36-second intervals. For example, in a bucket,
+  the measurements look like:
+  {t: 2023-01-01T5:00:00, m: 42, ...},
+  {t: 2023-01-01T5:00:36, m: 42, ...},
+  ...
+  {t: 2023-01-01T5:59:24, m: 42, ...}
+
+GlobalDefaults:
+  dbname: &db test
+  coll: &coll Collection0
+  nSensors: &nSensors 1000
+  nBucketsPerSensor: &nBucketsPerSensor 100
+  # (One-hour time span per bucket (3600 seconds)) / (36000 milliseconds (36 seconds) increment per
+  # measurement) = 100 measurements per bucket.
+  nMeasurementsPerBucket: &nMeasurementsPerBucket 100
+  nTotalMeasurements: &nTotalMeasurements
+    ^NumExpr:
+      withExpression: "nSensors * nBucketsPerSensor * nMeasurementsPerBucket"
+      andValues: {
+        nSensors: *nSensors,
+        nBucketsPerSensor: *nBucketsPerSensor,
+        nMeasurementsPerBucket: *nMeasurementsPerBucket
+      }
+  MaxPhases: &MaxPhases 4
+
+CreateTimeSeriesCollection:
```

Comment:
Here we define phases. So, can we rename those phases to `ABCPhase` so that they are distinguished from actor names?